src/main/scala/is/hail/expr/ir/MatrixIR.scala

```diff
@@ -610,12 +610,14 @@ case class MatrixAggregateRowsByKey(child: MatrixIR, expr: IR) extends MatrixIR
 
     val nCols = prev.nCols
 
+    val (minColType, minColValues, rewriteIR) = PruneDeadFields.pruneColValues(prev, expr)
```

Comment:
do you tend to use the prefix `min` or `new` or `pruned` elsewhere?

Reply:
I think it's `min` in a lot of places. It's a synonym with `pruned`